### DISCUSSION TITLE
Is this the latest version Version:  v0.107.25?

### DISCUSSION THREAD

**Discussion by CaptainComeback** [AdguardTeam/AdGuardHome#5635] [Q&A]
Hello I am using Portainer and it shows the image details and the Sha-256 sum does not match  what it says in the release. notes. Is this because in the release notes the sum is for a compressed image and inside Portainer the image is not compressed and if that is the case how would you verify? Thank you and sorry for asking so many questions.

**Comment by ainar-g** [comment]
See the [releases page](https://github.com/AdguardTeam/AdGuardHome/releases).  The latest release is v0.107.26.
